### PR TITLE
Added json config support

### PR DESCRIPTION
**Pull request by psychomario**
This patch allows arguments to be set in a json file.

Json arguments win over command line arguments, this is the wrong way around, but I'm not sure how to implement this cleanly.

Yaml would be nicer, but json is in base python.

### CODE REVIEW

**Comment by psychomario** [comment]
@mmattioli I'm not sure if you get PR notifications, but I would appreciate your comments on this.

File: pypxe-server.py [hunk 1]
```diff
@@ -29,6 +33,14 @@
 DHCP_BROADCAST = '<broadcast>'
 DHCP_FILESERVER = '192.168.2.2'
 
+#Service bools
```
Comment (mmattioli) [inline pypxe-server.py]: I'm not sure I understand why these lines were added, I thought the defaults were handled in `add_argument`?

Comment (psychomario) [inline pypxe-server.py]: They are, it's a relic from the first thing I tried. I'll remove it.

**Comment by psychomario** [comment]
Somehow deleted the remote branch. See other PR

**Comment by mmattioli** [comment]
@psychomario I just got around to reviewing the code for this and it looks fine albeit some minor comment cleanliness and string formatting but we can leave those for now and clean them all up when we're done reviewing the rest of the code.

**Pull request closed without merging**